.gitignore

```diff
@@ -1,3 +1,10 @@
+# Ignore frontend build related files
+node_modules
+timesketch/ui/static/dist
+
+# Ignore data files
+*.plaso
```

Comment:
You can move this to your own ignore:
.git/info/exclude

Reply:
".plaso" in .gitignore is useful for anyone using vagrant, so if you don't mind I will leave it here. Gitignore does not affect files already checked-in, so it will not break example timeline (vagrant/test.plaso)

Reply:
Sound good.